src/rust/engine/src/externs/interface.rs

```diff
@@ -169,8 +169,6 @@ impl PyTypes {
     digest_entries: &PyType,
     path_globs: &PyType,
     merge_digests: &PyType,
-    add_prefix: &PyType,
-    remove_prefix: &PyType,
```

Comment:
An example of why I think this change is valuable: we'll be able to get rid of `PyTypes` and `externs::unsafe_call()`. Do everything natively in Rust.